### PR TITLE
react-helmet pr

### PR DESCRIPTION
**Pull request by brunodhr**

### CODE REVIEW

File: src/screens/company/home.company.screen.tsx [hunk 1]
```diff
@@ -9,11 +9,12 @@ import {
...
-import {IReduxState} from 'reducers';
-import {IUserMeta} from 'types/api.type';
-import NotAuthorisedScreen from 'screens/403.screen';
-import NoticeBoard from 'components/noticeBoard';
+import {IDispatchFunction} from '../../types/common.type';
+import {IReduxState} from '../../reducers';
+import {IUserMeta} from '../../types/api.type';
+import NotAuthorisedScreen from '../403.screen';
+import NoticeBoard from '../../components/noticeBoard';
+import SEO from '../../components/seo'
```
Comment (Faisal-Manzer) [inline src/screens/company/home.company.screen.tsx]: We are moving to absolute imports. See issue #14

File: src/screens/student/home.student.screen.tsx [hunk 2]
```diff
@@ -186,31 +187,31 @@ const CourseApplication: FC<IProps> = () => {
...
-          <br />
-          <CourseApplication />
-        </Col>
-      </Row>
-    </div>
-  );
-};
+const HomeStudentScreen: FC<IProps> = () => (
+  <div className='container'>
+    <SEO title='Home' description='Home Studant Screen' />
```
Comment (Faisal-Manzer) [inline src/screens/student/home.student.screen.tsx]: Spelling correction.

**Comment by brunodhr** [comment]
> Change to absolute imports.
> Make the description more descriptive.

Oh right, I changed to absolute imports, but i'm not good with descriptions screens haha
any idea?

**Comment by Faisal-Manzer** [comment]
Hmmm, 🤔 actually I too am bad at writing a description.
You write simple one-liner desc I will merge ❤️

**Comment by Faisal-Manzer** [comment]
@brunodhr we added you as a contributor in README. Cheers 🍻

**Pull request merged**